linear/linear.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main(){
+	var a, b float64
+	fmt.Println("The program solves a linear inequality Ax + B > 0.Enter A and B: ")
```

Comment:
What if `A == 0`? The program outputs `result can't be found`. This is not true. The roots are either all real numbers, or no roots at all. Your program should handle this case.